sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/info/GetSamplesInfoInput.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.business.samples.info
+
+import life.qbic.datamodel.samples.Status
+
+/**
+ * <b>Input interface for the {@link GetSamplesInfo} feature</b>
+ *
+ * <p>Provides methods to trigger the fetching of sample codes and names of samples with provided status for one project</p>>
+ *
+ * @since 1.0.0
+ */
+interface GetSamplesInfoInput {
+
+    /**
+     * This method calls the output interface with the codes and names of samples in the project
+     * that have a certain status
+     * In case of failure the output interface failure method is called.
```

Comment:
```suggestion
     * <p>This method calls the output interface with the codes and names of samples in the project
     * that have a certain status</p>
     * <p>In case of failure the output interface failure method is called.</p>
```
Line breaks are not taken into consideration by JavaDoc. Please use the HTML tags if you want the second line to be in it's own line.